### PR TITLE
Read csv file and pseudocode

### PR DESCRIPTION
**Pull request by michaelmurwayi**

### CODE REVIEW

File: readfile.py [hunk 1]
```diff
@@ -1,28 +1,31 @@
...
+                    item["First_Name"] = row[0]
+                    item["Last_Name"] = row[1]
+                    item["Account_Name"] = row[2]
+                    item["Account_Number"] = row[3]
+                    item["Account_Type"] = row[4]
+                    item["Branch_Name"] = row[5]
+                    item["PhoneNumber"] = row[6]
+                    return Account
+
+print(ReadFile("accounts.csv"))
```
Comment (lincmba) [inline readfile.py:31]: Could we call this as a script `if __name__ == "__main__":`

File: readfile.py [hunk 2]
```diff
@@ -1,28 +1,31 @@
...
+            for row in reader:     
+                for item in Account:
+                    item["First_Name"] = row[0]
+                    item["Last_Name"] = row[1]
+                    item["Account_Name"] = row[2]
+                    item["Account_Number"] = row[3]
+                    item["Account_Type"] = row[4]
+                    item["Branch_Name"] = row[5]
+                    item["PhoneNumber"] = row[6]
+                    return Account
```
Comment (lincmba) [inline readfile.py:29]: Return after all the rows have been processed

File: tests.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import unittest
+import csv
+import readfile
+
+class TestReadFile(unittest.TestCase):
+    def test_ReadFile(self)-> None:
```
Comment (lincmba) [inline tests.py]: Add test for multiple rows of data

File: readfile.py (hunk 2 above)
Comment (ukanga) [inline readfile.py:29]: This only processes the first rows and then return it. Should it be creating a list of accounts or a list of only one account?

**Pull request closed without merging**